helm/stunner-gateway-operator/values.yaml

```diff
@@ -32,11 +32,14 @@ stunnerGatewayOperator:
           - --metrics-bind-address=127.0.0.1:8080
           - --leader-elect
           - --zap-log-level=info
+        securityContext:
+          allowPrivilegeEscalation: false
```

Comment:
setting the [original values](https://github.com/l7mp/stunner-helm/blob/72ac8bb9038a29e6701c8f1c3efd6a2f8785991b/helm/stunner-gateway-operator/templates/stunner-gateway-operator.yaml#L385-L386) as the default. can just set an empty object if there's no concern about breaking changes.

Reply:
Let's keep this as is, please.